wsd/FileServer.cpp

```diff
@@ -872,7 +873,8 @@ namespace
 void FileServerRequestHandler::preprocessFile(const HTTPRequest& request,
                                               const RequestDetails &requestDetails,
                                               Poco::MemoryInputStream& message,
-                                              const std::shared_ptr<StreamSocket>& socket)
+                                              const std::shared_ptr<StreamSocket>& socket,
+                                              std::map<std::string, std::string> replaceVar)
```

Comment:
Passing the argument as reference (`std::map<std::string, std::string>& replaceVar`) is better.